src/store/goods/reducer.ts

```diff
@@ -0,0 +1,46 @@
+import { createAsyncThunk, createSlice } from "@reduxjs/toolkit";
+import { getGoods } from "../../api";
+import { Good, LOAD_STATUSES_TYPES } from "../../types"
+
+const SLICE_NAME = "goods";
+
+const goodsOnBack = createAsyncThunk(SLICE_NAME, getGoods)
+
+
+interface GoodsStore {
+       loadStatus: string;
```

Comment:
у нас есть enum для типизации лоадстатусов
```suggestion
       loadStatus: LOAD_STATUSES_TYPES;
```